web/src/components/tasks/business-formation/ReviewSection.tsx

```diff
@@ -0,0 +1,261 @@
+import { Content } from "@/components/Content";
+import { Button } from "@/components/njwds-extended/Button";
+import { BusinessFormationDefaults } from "@/display-defaults/roadmap/business-formation/BusinessFormationDefaults";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { scrollToTop, setHeaderRole } from "@/lib/utils/helpers";
+import dayjs from "dayjs";
+import React, { ReactElement, useContext } from "react";
+import { FormationContext } from "../BusinessFormation";
+import { BusinessNameAndLegalStructure } from "./BusinessNameAndLegalStructure";
+
+export const ReviewSection = (): ReactElement => {
+  const { state, setTab } = useContext(FormationContext);
+  const { userData } = useUserData();
+
+  const headerLevelTwo = setHeaderRole(2, "h3-styling");
+
+  const getBusinessSuffixAndStartDate = () => {
+    return (
+      <>
+        <div className="display-block tablet:display-flex">
+          <div className="text-bold width-11rem">
+            <Content>{state.displayContent.reviewPageBusinessSuffix.contentMd}</Content>
+          </div>
+          <div>
+            {userData?.profileData.businessName} {state.formationFormData.businessSuffix}
+          </div>
+        </div>
+        <div className="display-block tablet:display-flex margin-top-1">
+          <div className="text-bold width-11rem">
+            <Content>{state.displayContent.reviewPageBusinessStartDate.contentMd}</Content>
+          </div>
+          <div>{dayjs(state.formationFormData.businessStartDate, "YYYY-MM-DD").format("MM/DD/YYYY")}</div>
+        </div>
+        <hr className="bg-base-lighter margin-y-205" />
+      </>
+    );
+  };
+
+  const getMainBusinessLocation = () => (
+    <>
+      <div className="flex space-between">
+        <div className="maxw-mobile-lg margin-bottom-2">
+          <Content overrides={{ h3: headerLevelTwo }}>
+            {state.displayContent.reviewPageLocation.contentMd}
+          </Content>
+        </div>
+        <div className="margin-left-2">
+          <Button
+            style="tertiary"
+            onClick={() => {
+              setTab(1);
+              scrollToTop();
+            }}
+            underline
+            dataTestid="edit-location-section"
+          >
+            {BusinessFormationDefaults.editButtonText}
+          </Button>
+        </div>
+      </div>
+      <div className="display-block tablet:display-flex">
+        <div className="text-bold width-11rem">
+          <Content>{state.displayContent.reviewPageAddress.contentMd}</Content>
+        </div>
+        <div>
+          {state.formationFormData.businessAddressLine1}
+          {", "}
+          {state.formationFormData.businessAddressLine2
+            ? `${state.formationFormData.businessAddressLine2}, `
+            : ""}
+          {userData?.profileData.municipality?.name}
+          {", "}
+          {state.formationFormData.businessAddressState} {state.formationFormData.businessAddressZipCode}
```

Comment:
I think that Mike's Members commit added a function to format an address into a single string. Could we reuse that here? Or in either case, turn that into a helper function?

Reply:
created helper function